guides/source-maps.md

```diff
@@ -248,6 +248,32 @@ minified\_url
 :   The full URL of the minified file. Should start with `http:` or
     `https:`
 
+### Using Source Maps On Many Domains
+
+If you'd like to use source maps with the same code that is deployed on many domains, use the following code, and be sure that the `minified_url` param when uploading your source maps includes 'dynamichost':
```

Comment:
Can we be more explicit about what to do with `dynamichost`? From reading this it just sounds like it needs to be present in the minified_url, when really, we need the customer to replace their hostname with that literal string. An example of the upload curl command would help.